tests/unit/s2n_pq_kem_hybrid_kat_test.c

```diff
@@ -0,0 +1,123 @@
+    const char *kat_file;
+    uint32_t server_key_msg_len;
+    uint32_t client_key_msg_len;
+    bool (*asm_is_enabled)();
+    S2N_RESULT (*enable_asm)();
+    S2N_RESULT (*disable_asm)();
+};
+
+static const struct s2n_kem_hybrid_test_vector test_vectors[] = {
+        {
+                .kem = &s2n_bike1_l1_r1,
+                .cipher_suite = &s2n_ecdhe_bike_rsa_with_aes_256_gcm_sha384,
+                .security_policy_name = "KMS-PQ-TLS-1-0-2019-06",
+                .kat_file = "kats/hybrid_ecdhe_bike_r1.kat",
+                .server_key_msg_len = 2875,
+                .client_key_msg_len = 2610,
+                .asm_is_enabled = s2n_pq_no_asm_available,
+                .enable_asm = s2n_pq_noop_asm,
+                .disable_asm = s2n_pq_noop_asm,
+        },
+        {
+                .kem = &s2n_bike1_l1_r2,
+                .cipher_suite = &s2n_ecdhe_bike_rsa_with_aes_256_gcm_sha384,
+                .security_policy_name = "KMS-PQ-TLS-1-0-2020-02",
+                .kat_file = "kats/hybrid_ecdhe_bike_r2.kat",
+                .server_key_msg_len = 3279,
+                .client_key_msg_len = 3014,
+                .asm_is_enabled = s2n_pq_no_asm_available,
+                .enable_asm = s2n_pq_noop_asm,
+                .disable_asm = s2n_pq_noop_asm,
+        },
+        {
+                .kem = &s2n_sike_p503_r1,
+                .cipher_suite = &s2n_ecdhe_sike_rsa_with_aes_256_gcm_sha384,
+                .security_policy_name = "KMS-PQ-TLS-1-0-2019-06",
+                .kat_file = "kats/hybrid_ecdhe_sike_r1.kat",
+                .server_key_msg_len = 711,
+                .client_key_msg_len = 470,
+                .asm_is_enabled = s2n_pq_no_asm_available,
+                .enable_asm = s2n_pq_noop_asm,
+                .disable_asm = s2n_pq_noop_asm,
+        },
+        {
+                .kem = &s2n_sike_p434_r2,
+                .cipher_suite = &s2n_ecdhe_sike_rsa_with_aes_256_gcm_sha384,
+                .security_policy_name = "KMS-PQ-TLS-1-0-2020-02",
+                .kat_file = "kats/hybrid_ecdhe_sike_r2.kat",
+                .server_key_msg_len = 663,
+                .client_key_msg_len = 414,
+                .asm_is_enabled = s2n_sikep434r2_asm_is_enabled,
+                .enable_asm = s2n_try_enable_sikep434r2_asm,
+                .disable_asm = s2n_disable_sikep434r2_asm,
+        },
+        {
+                .kem = &s2n_kyber_512_r2,
+                .cipher_suite = &s2n_ecdhe_kyber_rsa_with_aes_256_gcm_sha384,
+                .security_policy_name = "KMS-PQ-TLS-1-0-2020-07",
+                .kat_file = "kats/hybrid_ecdhe_kyber_r2.kat",
+                .server_key_msg_len = 1133,
+                .client_key_msg_len = 804,
+                .asm_is_enabled = s2n_pq_no_asm_available,
+                .enable_asm = s2n_pq_noop_asm,
+                .disable_asm = s2n_pq_noop_asm,
+        },
+};
+
+int main() {
+    BEGIN_TEST();
+    EXPECT_SUCCESS(s2n_disable_tls13());
```

Comment:
Since this is a new test, I'd try to write it without disabling TLS1.3. S2N now runs with TLS1.3 enabled by default but turned on/off via security policies.

Reply:
Oops, not sure why that was there at all.